src/main/native/jni_directbuffercompress_zstd.c

```diff
@@ -84,7 +84,7 @@ JNIEXPORT jlong JNICALL Java_com_github_luben_zstd_ZstdDirectBufferCompressingSt
     jclass dict_clazz = (*env)->GetObjectClass(env, dict);
     jfieldID compress_dict = (*env)->GetFieldID(env, dict_clazz, "nativePtr", "J");
     ZSTD_CDict* cdict = (ZSTD_CDict*)(intptr_t)(*env)->GetLongField(env, dict, compress_dict);
-    if (cdict == NULL) return ZSTD_ERROR(dictionary_wrong);
+    if (cdict == NULL) return ((size_t)-ZSTD_error_dictionary_wrong);
```

Comment:
should we declare our own ZSTD_ERROR macro - it looks more readable than doing the same calculation all over the place?